dimagi/utils/couch/pagination.py

```diff
@@ -258,7 +258,7 @@ def get_ajax_response(self, request, search_query, extras={}):
 
         to_return = {"sEcho": params.echo,
                      "iTotalDisplayRecords": total_rows,
-                     "aaData": all_json}
+                     "aaData": list(all_json)}
         
         for key, val in extras.items():
             to_return[key] = val
```

Comment:
Not related to your change, @czue, but I'm wondering whether setting the default value of extras to `{}` could potentially cause [problems][1]. 

  [1]: http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments